lib/board.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class Board
```

Comment:
**[OPTIONAL SUGESTION]**
Bring all 4 checks methods to the board class. Let the board check itself. This will make your code more organized following the OOP principle. This way the game class will just use the board object inside the determine_winner method to check if there is a winner.